src/modules/signer/signer.types.ts

```diff
@@ -33,3 +36,5 @@ export type AccountInfo = {
 
 export const PUBLIC_KEY = 'PublicKey';
 export const IS_ETH_SIGNER = 'isEthSigner';
+
+export type SignerT = Signer & TypedDataSigner;
```

Comment:
What is the relationship between `Signer` and `TypedDataSigner`? Could we just use `TypedDateSigner`?

What is the `T` meant to mean in `SignerT`?

Reply:
I couldn't find a good name for this type, so for now I left it as `SignerT`. Here is the code of the `TypedDataSigner`
```
export interface TypedDataSigner {
    _signTypedData(domain: TypedDataDomain, types: Record<string, Array<TypedDataField>>, value: Record<string, any>): Promise<string>;
}
``` 
And here is a comment for it in ethers lib 
```
// @TODO: This is a temporary measure to preserse backwards compatibility
//        In v6, the method on TypedDataSigner will be added to Signer
```